packages/config/src/Breakpoints.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * List of media queries.
+ * If max-width media query will achieve one of keys in list, then biggest will be returned
+ * In case window width is smaller then smallest key then default`s key value will be returned
+ * If no default value is set, then `1` will be returned
+ * @key Window width
+ * @value Number will be returned 
+ */
+export type Breakpoints = {
+  default?: number
```

Comment:
@Bitaru I don't think that we need default value here. We will define default map { width: columnsNumber } for new merchants

Reply:
@timakoder good point, so i will rewrite react hook to be like `useMedia({}, default_value)`